test/test_datasets.py

```diff
@@ -1801,5 +1801,87 @@ def test_targets(self):
                 assert item[6] == i // 3
 
 
+class LFWPeopleTestCase(datasets_utils.DatasetTestCase):
+    DATASET_CLASS = datasets.LFWPeople
+    FEATURE_TYPES = (PIL.Image.Image, int)
+    ADDITIONAL_CONFIGS = datasets_utils.combinations_grid(
+        split=('10fold', 'train', 'test'),
+        image_set=('original', 'funneled', 'deepfunneled')
+    )
+    _IMAGES_DIR = {
+        "original": "lfw",
+        "funneled": "lfw_funneled",
+        "deepfunneled": "lfw-deepfunneled"
+    }
+    _split = {'10fold': '', 'train': 'DevTrain', 'test': 'DevTest'}
```

Comment:
This could get a more expressive name, since this doesn't get you a split back, but takes one. Mabye 
```suggestion
    _file_id = {'10fold': '', 'train': 'DevTrain', 'test': 'DevTest'}
```